tests/e2e/privacy.spec.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+import { browser, expect, $, $$ } from '@wdio/globals';
+import {
+  enableExtension,
+  getExtensionElement,
+  getExtensionPageURL,
+  switchToPanel,
+} from './utils.js';
+
+async function updatePrivacySettings(name, value) {
+  await browser.url(await getExtensionPageURL('settings'));
+
+  const toggle = await getExtensionElement(`toggle:${name}`);
+  if ((await toggle.getProperty('value')) !== value) {
+    await toggle.click();
+  }
+
+  await browser.pause(1000);
```

Comment:
this may not be long enough for the DNR to update. but I understand that currently we have no visible side effect to wait for